tests/end_to_end/test_netcat/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,34 @@
+- name: Truncate files
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Truncate file alert.json
+      shell: echo "" > /var/ossec/logs/alerts/alerts.json
+      become: true
+
+- name: Generate events
+  hosts: wazuh-agent
+  ignore_errors: true
+  tasks:
+    - name: "{{ event_description_netcat }}"
+      command: "{{ command_netcat }}"
+      timeout: 30
+
+    - name: Wait for alert
+      pause:
+        seconds: 5
```

Comment:
```suggestion
    - name: Wait for alert
      wait_for:
        timeout: 5
```

Reply:
Done in a0f66312d81e5ec991fd2d28e1d4d0e92b922162

Reply:
Done in 96016ae9ca38441b9392f168996d51e0cf914dc7